content/03.results.md

```diff
@@ -114,69 +121,73 @@ Table 1: Molecular subtypes determined across OpenPBTA samples.
 
 #### Embryonal tumors
 **Figure {@fig:Fig2}B** shows the mutational landscape for 128 primary embryonal tumors. 
-The majority (N = 95) are medulloblastomas and span the spectrum of molecular subtypes: WNT, SHH, Group3, and Group 4 (see **Molecular Subtyping of CNS Tumors**), with their canonical mutations. 
+The majority (N = 95) of samples were medulloblastomas that spanned the spectrum of molecular subtypes (WNT, SHH, Group3, and Group 4; see **Molecular Subtyping of CNS Tumors**), as identified by subtype-specific canonical mutations. 
 We detected canonical _SMARCB1/SMARCA4_ deletions or inactivating mutations in atypical teratoid rhabdoid tumors (ATRTs) and C19MC amplification in the embryonal tumors with multilayer rosettes (ETMRs) [@doi:10.1007/s00401-020-02182-2; @doi:10.1093/neuonc/noab178; @doi:10.1186/s40478-020-00984-9; @doi:10.1038/nature22973].
 
 #### Diffuse astrocytic and oligodendroglial tumors
-In **Figure {@fig:Fig2}C**, we show genomic alterations in diffuse midline gliomas (DMGs, N = 34) and non-midline high-grade gliomas (N = 26) biopsied at diagnosis. 
-The single oligodendroglioma sample in the OpenPBTA does not contain mutations in the genes shown and is therefore not present in this oncoprint. 
-Across HGGs, we found _TP53_ (57%, 35/61) and _H3F3A_ (52%, 32/61) to be the most mutated and co-occurring genes (**Figure {@fig:Fig2}A**), followed by frequent mutations in _ATRX_ (30% 18/61). 
+In **Figure {@fig:Fig2}C**, we show genomic alterations in diffuse midline gliomas (DMGs, N = 34) and non-midline high-grade gliomas (HGGs, N = 26) biopsied at diagnosis. 
+The single oligodendroglioma sample (N = 1) in the OpenPBTA does not contain mutations in the genes shown and is therefore shown in this oncoprint. 
+Across diffuse astrocytic and oligodendroglial tumors, we found that _TP53_ (57%, 35/61) and _H3F3A_ (52%, 32/61) were both most mutated and co-occurring genes (**Figure {@fig:Fig2}A**), followed by frequent mutations in _ATRX_ (30% 18/61). 
 We found recurrent amplifications and fusions in _EGFR_, _MET_, _PDGFRA_, and _KIT_, highlighting that these tumors utilize multiple oncogenic mechanisms to activate tyrosine kinases, as has been previously reported [@doi:10.1002/ijc.32258; @doi:10.1016/j.ccell.2017.08.017; @doi:10.1186/s40478-020-00905-w]. 
-Gene set enrichment analysis showed upregulation (ANOVA p < 0.01) of DNA repair, G2M checkpoint, and MYC pathways as well as downregulation of the TP53 pathway (**Figure {@fig:Fig5}B**).
-The two tumors with ultra-high tumor mutation burden (TMB) (> 100 Mutations/Mb) were from patients with known mismatch repair deficiency syndrome [@doi:10.1093/neuonc/noz192].
+Gene set enrichment analysis showed upregulation (ANOVA p < 0.01) of DNA repair, G2M checkpoint, and MYC pathways as well as downregulation of the TP53 pathway (**Figure {@fig:Fig5}B**). **TODO: if we want to reference a figure 5 panel _here_, we need to moe the panel. Otherwise figures are out of order.**
```

Comment:
(move not moe)

Reply:
I don't necessarily think we need to move them per se - this is a challenge because GSVA is really a functional result of the somatic alterations, so it makes sense to back up the oncoprints with this data (I was asked about this for the PPTC paper before submitting), but GSVA can also stand alone with additional findings. So, it is tough - some people make some pie or bar plots with pathways or genes upregulated to go along with oncoprints, but it is hard to squeeze everything in, so this is what I had defaulted to.

Reply:
For now I'm going to remove this comment but open an issue so we can separately discuss this figure organization aspect.